hedera-mirror-rosetta/app/domain/types/address_book_entry_test.go

```diff
@@ -0,0 +1,49 @@
+package types
+
+import (
+	"github.com/coinbase/rosetta-sdk-go/types"
+	entityid "github.com/hashgraph/hedera-mirror-node/hedera-mirror-rosetta/app/domain/services/encoding"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func exampleAddressBookEntries() *AddressBookEntries {
+	return &AddressBookEntries{
+		[]*AddressBookEntry{
+			{
+				PeerId: &Account{
+					entityid.EntityId{
+						ShardNum:  0,
+						RealmNum:  0,
+						EntityNum: 0,
+					},
+				},
+				Metadata: nil,
```

Comment:
I think that it will be beneficial to check that `Metadata` properties are populated as-well. Not only the `PeerId`, because we are populating the IP and Port there:
```Go
map[string]interface{}{
    "ip":   e.Ip,
	"port": e.Port,
}
```
as-well